packages/sdk/src/services/item-service.ts

```diff
@@ -141,11 +163,7 @@ export class ItemService {
     this.logger.log('Reading file');
 
     this.logger.log('Fetching item');
-    const itemFetchResult = await this.get(
-      itemId,
-      auth.vault_access_token,
-      auth.data_encryption_key
-    ).catch(err => {
+    const itemFetchResult = await this.get(itemId, auth).catch(err => {
       throw new MeecoServiceError(
         `Unable to find item '${itemId}' - please check that the item exists for the current user.`
```

Comment:
This isn't part of the changes but I'm pointing it out now while I see it to fix later - we probably should actually be checking the status code here before blindly throwing "unable to find item".
It should only throw this error if it's literally a 404.

Reply:
fixed